__e2e__/config.test.ts

```diff
@@ -0,0 +1,45 @@
+import path from 'path';
+import {
+  runCLI,
+  getTempDirectory,
+  cleanup,
+  writeFiles,
+  spawnScript,
+} from '../jest/helpers';
+
+const cwd = getTempDirectory('test_root');
+
+beforeAll(() => {
+  // Register all packages to be linked
+  for (const pkg of ['platform-ios', 'platform-android']) {
+    spawnScript('yarn', ['link'], {
+      cwd: path.join(__dirname, `../packages/${pkg}`),
+    });
+  }
+
+  // Clean up folder and re-create a new project
+  cleanup(cwd);
+  writeFiles(cwd, {});
+
+  // Initialise React Native project
+  runCLI(cwd, ['init', 'TestProject']);
```

Comment:
We avoid running init with default template in e2e tests, because it's flaky. Can you init from a local template like in `__e2e__/init.test.ts`?

Reply:
On the second thought, let's give it a chance, maybe it won't be flaky this time.